Phase05/Phase05-Console/Phase05-Console/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿using InvertedIndex;
+
+namespace Phase05_Console;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        var path = @"C:\Users\Arya\OneDrive\Desktop\are\Summer1401-SE-Team08\Phase05-Console\EnglishData";
```

Comment:
this code only works on one system because of the absolute path
try using a relative path instead